docker/prepare-dev-environment/solr/docker-compose.yml

```diff
@@ -0,0 +1,113 @@
+version: "3.6"
+
+services:
+  solr-populator:
+    build: ${DOCKERFILE_PATH}
+    networks:
+      - atlas-test-net
+    depends_on:
+      - postgres
+      - solrcloud-0
+      - solrcloud-1
+    volumes:
+      - gradle-wrapper-dists:/root/.gradle/wrapper/dists
+      - gradle-ro-dep-cache:/gradle-ro-dep-cache:ro
+      - atlas-data-bioentity-properties:/atlas-data/bioentity_properties
+      - atlas-data-exp:/atlas-data/exp
+      - atlas-data-expdesign:/atlas-data/expdesign
+    environment:
+      JAVA_TOOL_OPTIONS: "-Dfile.encoding=UTF8"
+      EXP_IDS: ${EXP_IDS}
```

Comment:
Do we need to redefine the env variables that was sourced from the dev.env file?

Reply:
Removed!